content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -437,29 +437,30 @@ HD/Library/LaunchDaemons" file's _$JENKINS_HOME_ property.
 By default, the _$JENKINS_HOME_ will be set to "Macintosh
 HD/Users/Shared/Jenkins".
 
-**Ubuntu/Debian**
+**Linux**
 
-If you install Jenkins using a Debian package, you can find and edit the
-location of your _$JENKINS_HOME_ by editing your "/etc/default/jenkins" file.
+By default, `$JENKINS_HOME` is set to `/var/lib/jenkins`
+and `$JENKINS_WAR` is set to `/usr/share/java/jenkins.war`.
 
-By default, the _$JENKINS_HOME_ will set to "/var/lib/jenkins" and your
-$JENKINS_WAR will point to "/usr/share/jenkins/jenkins.war".
+You can edit the location of `$JENKINS_HOME`
+by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf`:
 
-**Red Hat/CentOS/Fedora**
-
-If you install Jenkins as a RPM package, the default _$JENKINS_HOME_ will be
-"/var/lib/jenkins".
-
-You can edit the location of your _$JENKINS_HOME_ by editing the
-"/etc/sysconfig/jenkins" file.
-
-**openSUSE**
+[source]
+----
+[Service]
+Environment="HOME=/var/lib/jenkins"
+Environment="JENKINS_HOME=/var/lib/jenkins"
+WorkingDirectory=/var/lib/jenkins
+----
 
-If installing Jenkins as a package using zypper, you'll be able to edit the
-_$JENKINS_HOME_ by editing the "/etc/sysconfig/jenkins" file.
+You can edit the location of `$JENKINS_WAR`
+by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf`:
```

Comment:
Also worth suggesting `systemd edit jenkins` here?
```suggestion
by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf` (for example, using `systemd edit jenkins`):
```

Reply:
Done in commit 4b280c8.